usmqe_tests/api/ceph/test_pool.py

```diff
@@ -0,0 +1,587 @@
+
+        Description
+        ===========
+
+        Check if there is created pool in ceph cluster via CLI.
+
+        .. test_step:: 2
+
+            Connect to ceph monitor machine via ssh and run
+            ``ceph --format json --cluster *clustername* osd pool ls detail``
+
+        .. test_result:: 2
+
+            There should be listed ceph pool named ``valid_pool_name``.
+
+            """
+# TODO get proper cluster name from configuration
+    storage = ceph_cluster.CephCluster(pytest.config.getini("usm_ceph_cl_name"))
+    pools = storage.osd.pool_ls(detail=True)
+    LOGGER.info("List of Ceph pools:{}".format(pools))
+    selected_pools = [pool for pool in pools
+                      if pool["pool_name"] == valid_pool_name
+                      ]
+    pytest.check(len(selected_pools) == 1,
+                 "Pool {} should be created in Ceph \
+                 cluster {}.".format(valid_pool_name,
+                                     pytest.config.getini("usm_ceph_cl_name"))
+                 )
+
+    """@pylatest api/ceph.create_pool_valid
+        API-ceph: create_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        Description
+        ===========
+
+        Check if there is created pool in ceph cluster via API.
+
+        .. test_step:: 3
+
+            Connect to Tendrl API via GET request
+            to ``APIURL/:cluster_id/CephPoolList``
+            Where cluster_id is set to predefined value.
+
+        .. test_result:: 3
+
+            There should be listed ceph pool named ``valid_pool_name``.
+
+            """
+
+    if len(pools) == 1:
+        pool = pools[0]
+        storage_pool_attributes = {
+            "erasure_code_profile": pool["erasure_code_profile"],
+            "min_size": pool["min_size"],
+            "percent_used": "0",  # newly created pool
+            "pg_num": pool["pg_num"],
+            "pool_id": pool["auid"],
+            "pool_name": pool["pool_name"],
+            "quota_enabled":
+                pool["quota_max_bytes"] == 0 and
+                pool["quota_max_objects"] == 0,
+            "quota_max_bytes": pool["quota_max_bytes"],
+            "quota_max_objects": pool["quota_max_objects"],
+            "size": pool["size"],
+            "type": "replicated" if pool["type"] == 1 else "ecpool",  # TODO
+            "used": "0"  # newly created pool
+            }
+
+        pool_tendrl = [pool_t for pool_t in api.get_pool_list(valid_cluster_id)
+                       if pool_t["pool_name"] == valid_pool_name
+                       ][0]
+        # remove Tendrl specific keys
+        pool_tendrl.pop("deleted")
+        pool_tendrl.pop("hash")
+        pool_tendrl.pop("updated_at")
+
+        pytest.check(
+            pool_tendrl == storage_pool_attributes,
+            """Storage pool attributes: {}
+            Tendrl pool attributes: {}
+            These should be the same.""".format(
```

Comment:
@ltrilety There is compare of 2 dictionaries and not lists. In case of dictionaries doesn't matter about item order. OK I'll remove those spaces.

@mbukatov You can see difference in this comment because both dictionaries are logged.